### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by refeg**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Welcome To Demo</title>
 </head>
 <body>
-    
+    <h1>Welcome To Demo ! Enjor your stay.</h1>
```
Comment (fehocir) [inline index.html:10]: could remove 'enjor your stay' string amd make welcome to demo read?

File: index.html [hunk 2]
```diff
@@ -7,6 +7,6 @@
...
 </body>
```
Comment (refeg) [inline index.html:11]: chenage rade

File: index.html (hunk 1 above)
Comment (refeg) [reply index.html:10]: sure can do

File: index.html (hunk 2 above)
Comment (fehocir) [reply index.html:11]: yes

**Pull request merged**